python/torch_mlir_e2e_test/test_suite/conv.py

```diff
@@ -99,6 +99,34 @@ def Conv2dBiasNoPaddingModule_basic(module, tu: TestUtils):
     t = tu.rand(5, 2, 10, 20)
     module.forward(t)
 
+# ==============================================================================
+
+class Convolution2DBFloat16(torch.nn.Module):
```

Comment:
I don't see bf16 in this test - am I missing something?

Reply:
You are not. I forgot to change the type. Thanks!

Reply:
Test removed since RefBackend doesn't support BF16.